src/databricks/labs/ucx/contexts/application.py

```diff
@@ -358,10 +358,14 @@ def notebook_loader(self) -> NotebookLoader:
     def notebook_resolver(self):
         return NotebookResolver(self.notebook_loader)
 
+    @cached_property
+    def site_packages_path(self):
+        lookup = self.path_lookup
+        return next(path for path in lookup.paths if "site-packages" in path.as_posix())
```

Comment:
We returning only the first site-packages? What should we do with (possible) later ones?

Reply:
It's a good question but maybe not in scope of this PR. Requires %pip and job task settings implementation to find out.